src/hevm/src/EVM/Expr.hs

```diff
@@ -265,6 +273,9 @@ copySlice (Lit srcOffset) (Lit dstOffset) (Lit size) (ConcreteBuf src) (Concrete
         tl = BS.drop (num dstOffset + num size) dst
     in ConcreteBuf $ hd <> sl <> tl
 
+-- copying 32 bytes can be rewritten to a WriteWord on dst (e.g. CODECOPY of args during constructors)
+copySlice srcOffset dstOffset (Lit 32) src dst = WriteWord dstOffset (readWord srcOffset src) dst
```

Comment:
We can also return `writeWord dstOffset (readWord srcOffset src) dst` here, that will perform some concrete execution if possible. When `copySlice` is used `simplify` it does matter, because this will get simplified in an other iteration, but it may matter if we ever use a standalone `copySlice`

Reply:
done